ndb/tests/unit/test_query.py

```diff
@@ -291,15 +291,36 @@ def test__to_filter():
             parameter_node._to_filter()
 
     @staticmethod
-    def test_resolve():
+    def test_resolve_simple():
         prop = model.Property(name="val")
         param = query.Parameter("abc")
         parameter_node = query.ParameterNode(prop, "=", param)
 
+        value = 67
+        bindings = {"abc": value}
         used = {}
-        with pytest.raises(NotImplementedError):
-            parameter_node.resolve({}, used)
-        assert used == {}
+        resolved_node = parameter_node.resolve(bindings, used)
+
+        assert resolved_node == query.FilterNode(b"val", "=", value)
```

Comment:
The property name gets converted to bytes?

Reply:
For now yes, but that was a Python 3 vs. Python 2 weirdness. I think at some point we'll need to dive in and figure out the differences but for now I just "copied" the old behavior.